examples/step-53/step-53.cc

```diff
@@ -42,9 +42,11 @@
 // some BOOST-provided functionality to read directly from gzipped data.
 #include <deal.II/base/function_lib.h>
 
+DEAL_II_DISABLE_EXTRA_DIAGNOSTICS
```

Comment:
Since it's a tutorial, I think it would be nice to explain what these extra lines do (in the comment above,line 42).